tests/helpers.bash

```diff
@@ -10,6 +10,15 @@ BUILD_NUM="${BUILD_NUMBER:-0}"
 JOB_BASE="${JOB_BASE_NAME:-local}"
 BUILD_ID="${JOB_BASE}-${BUILD_NUM}"
 
+# Prefer local build if binary file detected.
```

Comment:
If you make a change locally, build and then run a single test. Your changes are
not necessarily being tested.  Since the binary used is what you have in your
`$PATH` you will need to run `make install` before the test.  I have forgotten
todo that a lot and have ended up testing the wrong binary... By putting this
here we prefer to use the local binary when present. So this makes it easier to
test your local changes on single tests. Does it make sense?